policykit/policyengine/models.py

```diff
@@ -1148,9 +1162,26 @@ def save(self, *args, **kwargs):
 
         super(PlatformActionBundle, self).save(*args, **kwargs)
 
+class PlatformPolicyManager(models.Manager):
+    def get_queryset(self):
+        return super().get_queryset().filter(kind=Policy.PLATFORM)
+
+class ConstitutionPolicyManager(models.Manager):
+    def get_queryset(self):
+        return super().get_queryset().filter(kind=Policy.CONSTITUTION)
```

Comment:
this lets you do `Policy.platform_policies.all()` and `Policy.constitution_policies.all()`